templates/taxi/manufacturer_list.html

```diff
@@ -23,6 +25,12 @@ <h1>Manufacturer List
               <td>
                   {{ manufacturer.country }}
               </td>
+              <td>
+                <a href="{% url 'taxi:manufacturer-update' pk=manufacturer.id %}">Update</a>
+              </td>
+              <td>
+                <a href="manufacturer_confirm_delete.html">Delete</a>
```

Comment:
The 'Delete' link currently points to a static HTML file. Replace it with a Django URL template tag to point to the correct delete view, such as `{% url 'taxi:manufacturer-delete' pk=manufacturer.id %}`.